seed/views/v3/uploads.py

```diff
@@ -348,7 +349,7 @@ def create_from_pm_import(self, request):
             rows.append(this_row)
 
         # Then write the actual data out as csv
-        with open(path, 'w', encoding='utf-8') as csv_file:
+        with default_storage.open(path, 'w') as csv_file:
```

Comment:
Should this line keep `encoding='utf-8'`?

Reply:
Not really necessary, it works without the encoding.

Reply:
Okay! Can you keep the encoding anyways then?

Reply:
This causes issues and raises the following
```
TypeError: open() got an unexpected keyword argument 'encoding'
```

Reply:
Oh really? But you have the same line `with default_storage.open(path, 'w', encoding='utf-8') as csv_file:` in seed/data_importer/views.py, does that one throw an error as well?